include/morphio/enums.h

```diff
@@ -61,9 +61,15 @@ enum SectionType {
     SECTION_GLIA_PROCESS = 2,     // TODO: nasty overload there
     SECTION_GLIA_ENDFOOT = 3,
 
-    // All section types equal or above this number are custom types according
-    // to neuromorpho.org standard
-    SECTION_CUSTOM_START = 5,
+    SECTION_CUSTOM_5 = 5,
```

Comment:
You can add a comment here about the custom section

Reply:
please verify, let me know if it is not sufficient

Reply:
Yes, that's fine.